manifests/validate_db_connection.pp

```diff
@@ -47,7 +47,7 @@
   # time it takes to run each psql command.
   $timeout = (($sleep + 2) * $tries)
 
-  $exec_name = "validate postgres connection for ${database_host}/${database_name}"
+  $exec_name = "validate postgres connection for ${database_username}@${database_host}:${database_port}/${database_name}"
```

Comment:
`database_username` and `database_port` default to undef. We don't want to end up with stray `@` and `:` symbols in the exec